Documentation/ceph-teardown.md

```diff
@@ -32,10 +32,24 @@ kubectl delete -f csi/cephfs/kube-registry.yaml
 kubectl delete storageclass csi-cephfs
 ```
 
-## Delete the CephCluster CRD
-
 After those block and file resources have been cleaned up, you can then delete your Rook cluster. This is important to delete **before removing the Rook operator and agent or else resources may not be cleaned up properly**.
 
+## Automatic Cleanup
```

Comment:
How about if we just keep this in the same section? We're just recommending to use the automatic cleanup by default now.
```suggestion
## Delete the CephCluster CR
```